actors/builtin/multisig/multisig_actor.go

```diff
@@ -78,6 +78,10 @@ func (a Actor) Constructor(rt vmr.Runtime, params *ConstructorParams) *adt.Empty
 		rt.Abortf(exitcode.ErrIllegalArgument, "must have at least one signer")
 	}
 
+	if params.NumApprovalsThreshold > int64(len(params.Signers)) {
```

Comment:
A couple other cases we should consider checking:
- Negative `NumApprovalsThreshold`
- Negative `UnlockDuation`
- Duplicate `Signers` -- e.g. `signers = addr.Address{"alice", "bob", "bob", bob"}`

Reply:
Both the negative checks are taken care of by using cbor and int64 typed fields.  Our cbor decoder should fail reading params bytes into the ConstructorParams struct.  If we want to prohibit duplicate signers you should file another issue.  It's not clear to me that we want to enforce that and that design discussion should be separate from this PR.

Reply:
int64 means cbor doesn't help

Reply:
Changing to `uint64` in `ConstructorParams` is ok. The days of "int64 only" are passed. However, I think we actually want to require that approvals >= 1, so check that explicitly. Please also check the `UnlockDuration` and file an issue for validating signers.